infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -247,6 +247,62 @@ locals {
         "/v1.0/groups",
         "/v1.0/groups/{group-id}/members"
       ]
+    },
+    "msft-teams" : {
+      source_kind : "msft-teams"
+      worklytics_connector_id : "msft-teams-psoxy",
+      display_name : "Microsoft Teams"
+      identifier_scope_id : "azure-ad"
+      source_auth_strategy : "oauth2_refresh_token"
+      target_host : "graph.microsoft.com"
+      required_oauth2_permission_scopes : [],
+      required_app_roles : [
+        "Team.ReadBasic.All",
+        "Channel.ReadBasic.All",
+        "Chat.ReadBasic.All",
+        "Chat.Read.All",
+        "ChannelMessage.Read.All",
+        "CallRecords.Read.All",
+        "OnlineMeetings.Read.All"
```

Comment:
As now we support users, we'll need `User.Read.All` [scope](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#permissions)

Reply:
Done.